app/models/item.js

```diff
@@ -166,6 +170,26 @@ export default cloudinaryUrl.extend({
     }
   }),
 
+  hasSingleLocation: Ember.computed('packagesLocations.[]', function(){
+    if(this.get('packagesLocations')){
+      return this.get('packagesLocations').length === 1;
+    }
```

Comment:
@namrataukirde , Yes it is. A minor change suggestion. 
How about using `Ember.isEqual(this.get('packagesLocations.length'), 1)` or `Ember.isEqual(this.get('packagesLocations').length, 1)` instead of `Ember.isEqual(this.get('packagesLocations').get('length'), 1)`?